lib/iex/lib/iex.ex

```diff
@@ -419,17 +448,23 @@ defmodule IEx do
   Pries into the process environment.
 
   This is useful for debugging a particular chunk of code
-  and inspect the state of a particular process. The process
-  is temporarily changed to trap exits (i.e. the process flag
-  `:trap_exit` is set to `true`) and has the `group_leader` changed
-  to support ANSI escape codes. Those values are reverted by
-  calling `respawn/0`, which starts a new IEx shell, freeing up
-  the pried one.
-
-  When a process is pried, all code runs inside IEx and, as
-  such, it is evaluated and cannot access private functions
-  of the module being pried. Module functions still need to be
-  accessed via `Mod.fun(args)`.
+  when executed by a particular process. The process becomes
+  the evaluator of IEx commands and is temporarily changed to
+  have a custom group leader. Those values are reverted by
+  calling `IEx.Helpers.respawn/0`, which starts a new IEx shell,
+  freeing up the pried one.
+
+  When a process is pried, all code runs inside IEx and has
+  access to all imports and aliases from the original code.
+  However, the code is evaluated and therefore cannot access
+  private functions of the module being pried. Module functions
+  still need to be accessed via `Mod.fun(args)`.
+
+  Alternatively, you can use `IEx.break!/4` to setup a breakpoint
```

Comment:
I just wanted to give a quick run down of the pros and cons so the user doesn't have to figure it out for themselves.